pkgs/tools/system/undaemonize/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "undaemonize";
```

Comment:
add the date '<year>-<month>-<day>' of the commit you referring to here as version.